README.md

```diff
@@ -35,6 +35,11 @@ First generate a user model with a `hashed_password` field.
 $ mix phoenix.gen.model User users email hashed_password
 ```
 
+Please note: we recommend using [citext] for the email column so that your
```

Comment:
Thoughts on adding `(or equivalent for non-postgres db's)` or something similar?